src/components/StoryCreationComponent.js

```diff
@@ -12,45 +12,29 @@ import { Col, Row } from 'reactstrap';
 
 import '../styles/AttributeDeselector.css';
 
+//React quill
+import * as ReactQuill from 'react-quill'; // Typescript
+import 'react-quill/dist/quill.snow.css'; // ES6
+
 class StoryCreationComponent extends Component {
   constructor(props) {
     super(props);
 
     this.state = {
       isLoaded: false,
-      selectedAttributes: [],
       authorField: '',
       titleField: '',
       tagsInputValue: '',
       tagsField: [],
       storyField: '',
+      storyTextOnly: '',
       storyMaxLength: 1000
```

Comment:
This should be a constant, you can move it outside of this class:
`const MAX_LENGTH = 1000;`

Reply:
done